mne/preprocessing/tests/test_realign.py

```diff
@@ -113,7 +162,7 @@ def _assert_similarity(raw, other, n_events, events_raw=None):
     evoked_raw = Epochs(raw, events_raw, **kwargs).average()
     evoked_other = Epochs(other, events_other, **kwargs).average()
     assert evoked_raw.nave == evoked_other.nave == len(events_raw)
-    assert len(evoked_raw.data) == len(evoked_other.data) == 1  # just EEG
-    corr = np.corrcoef(evoked_raw.data[0], evoked_other.data[0])[0, 1]
-    assert 0.9 <= corr <= 1.0
+    assert len(evoked_raw.data) == len(evoked_other.data) == 2  # just EEG
+    # corr = np.corrcoef(evoked_raw.data[0], evoked_other.data[0])[0, 1]
+    # assert 0.9 <= corr <= 1.0 # may not hold when drift is large
```

Comment:
can we still test this but either use a lower threshold for large drifts, or at least do an `if`-clause so it still gets asserted for small drifts?

Reply:
Done. Although for the same reason as above, I don't think it's a meaningful test